packages/core/src/common/keybinding.ts

```diff
@@ -128,10 +128,15 @@ export class KeybindingRegistry {
     }
 
     /**
+     * The `checkAvailability` flag with `false` could come handy when we do not want to check whether the command is currently active or not.
+     * For instance, when building the main menu, it could easily happen that the command is not yet active (no active editors and so on)
+     * but still, we have to build the key accelerator.
+     *
      * @param commandId the unique ID of the command for we the associated ke binding are looking for.
+     * @param checkAvailability if `false` then the availability of the command will not be checked. Default is `true`
      */
-    getKeybindingForCommand(commandId: string): Keybinding | undefined {
-        return (this.commands[commandId] || []).find(binding => this.isValid(binding));
+    getKeybindingForCommand(commandId: string, checkAvailability: boolean = true): Keybinding | undefined {
```

Comment:
- naming should be consistent, i.e. the keybinding should be active similar to handlers, not available;
- the second arg can be of the object type, not the boolean type, to make the user code readable, e.g. 
`getKeybindingForCommand(id, { active: false })` instead of `getKeybindingForCommand(id, false)`, plus the new options can be added
  -  or to have 2 method instead of the second arg: `getActiveKeybidingForCommand` and `getKeybindingForCommand`